### DISCUSSION TITLE
problem with domain setting

### DISCUSSION THREAD

**Discussion by simonator1001** [coollabsio/coolify#3136] [Q&A]
Hi, so I've setup n8n and I've added a a record name n8n on my domian and have set n8n.my-domain.com in coolify n8n server, but when i click the link after deployment, it takes me back to the coolify dashboard instead of the n8n application, would anyone have any idea to how to solve this? Many thanks!

**Comment by peaklabs-dev** [comment]
Have you set the domain name correctly here?
<img width="787" alt="image" src="https://github.com/user-attachments/assets/0f1be74e-1556-4326-ba22-7d5c30d34d86">

**Comment by bscott** [comment]
Having this same issue and ensure I have a wild card domain set.

**Comment by zuhaili1986** [comment]
Same here, I've watched a few YouTube videos, but I still don't have a solution for the domain problem.

**Reply by djsisson** [reply]
could you give an example of what you have set for n8n

**Reply by zuhaili1986** [reply]
I just got my problem solved. It was related to my domain still not having an SSL certificate. My older domain, even with a domain purchase from Cloudflare, did not solve the problem. Setting the DNS for n8n.mydomain.com using Cloudflare still did not solve the problem, leading me to buy another domain from Hostinger.  Maybe Hostinger provides an SSL certificate; I am not sure. But when I used n8n.mydomain.com (purchased from Hostinger), it solved my issue with the domain not opening the n8n interface.

**Reply by djsisson** [reply]
if your domain on cloudflare was using the dns proxy, you need to install the cf origin certificate or use dns challenge, as you can't generate a certificate using http challenge.

**Reply by zuhaili1986** [reply]
Oh, okay, thanks for the clarification.🙏
